docs/integrations/llms/pydanticai.md

```diff
@@ -0,0 +1,68 @@
+---
+integration: logfire
+---
+
+## Introduction
+
+Logfire supports instrumenting [PydanticAI](https://ai.pydantic.dev/).
+
+```python hl_lines="15"
+import logfire
+from pydantic_ai import Agent, RunContext
+
+logfire.configure()
+
+
+roulette_agent = Agent(
+    'openai:gpt-4o',
+    deps_type=int,
+    result_type=bool,
+    system_prompt=(
+        'Use the `roulette_wheel` function to see if the '
+        'customer has won based on the number they provide.'
+    ),
+    instrument=True
+)
+
+
+@roulette_agent.tool
+async def roulette_wheel(ctx: RunContext[int], square: int) -> str:
+    """check if the square is a winner"""
+    return 'winner' if square == ctx.deps else 'loser'
+
+
+# Run the agent
+success_number = 18
+result = roulette_agent.run_sync('Put my money on square eighteen', deps=success_number)
+print(result.data)
+#> True
+
+result = roulette_agent.run_sync('I bet five is the winner', deps=success_number)
+print(result.data)
+#> False
+```
+
+Note:
+
+- We pass `instrument=True` to the agent.
+- You will need to install the logfire SDK extra for your LLM provider, e.g. `pip install "logifre[openai]"`
```

Comment:
@alexmojaki is this true?

Reply:
that extra just installs logfire, so not really

Reply:
ok, removed (and tested)